src/components/Icons/Finances/ChartColumUp.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function ChartColumUp({
+  id = 'ChartColumUpIcon',
```

Comment:
🔧 

```suggestion
function ChartColumnUp({
  id = 'ChartColumnUpIcon',
```

And the file name

Reply:
Good catch! 👌